webpack.config.js

```diff
@@ -0,0 +1,23 @@
+var path = require('path');
+var SRC_DIR = path.join(__dirname, '/client/src');
+var DIST_DIR = path.join(__dirname, '/client/dist');
+
+module.exports = {
+  entry: `${SRC_DIR}/index.jsx`,
+  output: {
+    filename: 'bundle.js',
+    path: DIST_DIR
+  },
+  module : {
+    loaders : [
+      {
+        test : /\.jsx?/,
+        include : SRC_DIR,
+        loader : 'babel-loader',
+        query: {
+          presets: ['react', 'es2015']
```

Comment:
Please do not copy/paste webpack configs. This is not a modern webpack config; please reference the official webpack and Babel documentation.

Reply:
https://github.com/FEC-HRR41-PICOLLO/djason-ikea-similar-products/pull/9